python/graphstorm/config/argument.py

```diff
@@ -1666,6 +1668,28 @@ def eval_metric(self):
 
         return eval_metric
 
+    @property
+    def lp_major_eval_etype(self):
+        """ Canonical etype used for selecting the best model
+        """
```

Comment:
this can only specify the evaluation on one edge type?

Reply:
how are we going to select multiple edge types?

Reply:
This specifies which edge type is used to select the model.
Currently it can be 1) ALL, using the average MRR of all edge types and 2) a specific edge type.

Reply:
evaluation is controlled by --eval-etypes